Communityhub/Script/Reset local Admin Password.ps1

```diff
@@ -0,0 +1,6 @@
+﻿Param(
+[Parameter(Mandatory=$True)]
+[string]$Password
+)
+
+net user administrator $password
```

Comment:
The contents of this script will cause the password to be placed on the command line on the target machines.  This means that it will be logged in the OS and would potentially be obtainable via WMI.  This is a security risk, and should not be considered a secure way to do this.